### DISCUSSION TITLE
Many To Many Relationships with Relation manager

### DISCUSSION THREAD

**Discussion by manythrough** [filamentphp/filament#4255] [Help]
Dear Community,

I would like to use the Relation manager to model a many to many relationship like the one presented in the laravel documentation here:
https://laravel.com/docs/9.x/eloquent-relationships#many-to-many

Specifically how to deal with the "role_user" table from the example.

Ideally, the user of the admin panel would pick a user and select a role for that user without knowing about the table in between.

Can this be done with the Relation manager?

**Comment by danharrin** [comment]
Yes, the relation manager can attach roles to the user without you having to do anything
